src/store/mutations.js

```diff
@@ -13,3 +13,15 @@ export const initializeNavigation = ({ workspace, workspaceNavigation }) => {
 
   workspaceNavigation.items = AppNavigation.assembleNavigationItems(topItems, workspace)
 }
+
+export const initializeData = (state) => {
+  const xobj = new XMLHttpRequest()
```

Comment:
All async action should be wrapped in actions, mutations only for sync operations 

For example
https://github.com/zorko-io/zorko-web-app/blob/master/src/store/actions.js#L3

Or you can keep it static and use webpack import to request json for example 

https://github.com/nesterone/vue-vega/blob/master/_docs/partials/examples/SignalInteractivityExample.vue#L10
If static doesn't work for now, just drop car's file under src/, webpack can import json by default, 

If you would use webpack, then you can go with mutator

Reply:
Understand, i`ll fix it.